### ISSUE TITLE
Build out fails with keystoneauth1 version issuie

### ISSUE THREAD

**Issue by HellDesk** [catalyst/cloud-pydashie#1]
Installed /usr/local/lib/python2.7/site-packages/Cloud_PyDashie-0.3.1-py2.7.egg
Processing dependencies for Cloud-PyDashie==0.3.1
error: keystoneauth1 2.19.0 is installed but keystoneauth1>=2.21.0 is required by set(['osc-lib'])
The command '/bin/sh -c python setup.py install' returned a non-zero code: 1

**Comment by Adrian-Turjak** [comment]
Yes! Ran into that one recently as we haven't redeployed this in a while.

Am going to dig into it and likely do a bit of a rewrite and update a bunch of the required versions.
